drivers/hmis_csv_twenty_twenty_four/app/models/hmis_csv_twenty_twenty_four/exporter/assessment/overrides.rb

```diff
@@ -25,6 +25,10 @@ def process(row)
 
     def self.apply_overrides(row)
       row = replace_blank(row, hud_field: :AssessmentLocation, default_value: 'unknown')
+      row = replace_newlines(row, hud_field: :AssessmentLocation)
+      row = replace_newlines(row, hud_field: :AssessmentQuestionGroup)
+      row = replace_newlines(row, hud_field: :AssessmentQuestion)
+      row = replace_newlines(row, hud_field: :AssessmentAnswer)
```

Comment:
Pretty sure these 3 need to go on assessment_question/overrides

Reply:
Whoops, thanks. Will fix.